### ISSUE TITLE
Menus: Add role="presentation" to the submenu buttons

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/newspack-theme#1279]
Right now, the primary and secondary menus can use buttons to indicate that a menu item has dropdowns. 

Because these buttons are empty, and don't serve anything other than a visual role, they should have the attribute `role="presentation"`.
